tutorials/howtos/forms.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Building a Contact Form with Apostrophe"
+layout: tutorial
+---
+
+# Building a Contact Form with Apostrophe
+
+So you want a contact form on your site---a pretty common requirement. Maybe it's not a contact form; maybe you want to accept submissions of stories, or product ideas. The same principles apply. Apostrophe provides several ways to accomplish this.
+
+## The easy way: `apostrophe-forms`
+
+The **[Apostrophe Forms module](https://www.npmjs.com/package/apostrophe-forms)** was more recently updated to work with modern Apostrophe projects. This is your simplest way to create forms in your projects. After installing and configuring the module as directed, you and your website users can create and update forms as you would any other piece of content. That form can be placed on any page or pages using the forms widget. This allows for a single form to live many places on a website if needed.
+
+The primary case where this is not desirable is if the website needs only one or two forms and those forms should *never* (or almost never) change. For example, if a form is very tightly coupled to a third-party API, it might be too risky to let website editors change the form. Even this could mostly be avoided by limiting access to forms with [user group permissions](https://docs.apostrophecms.org/apostrophe/tutorials/intermediate/permissions).
+
+In most every case, this is the recommended way to create forms for Apostrophe projects.
+
+## Doing it the old school way: using `apostrophe-pieces-submit-widgets`
```

Comment:
Please change to "Creating pieces via forms: apostrophe-pieces-submit-widgets"

It's still a real use case, it's not really old school.

Reply:
Updated